src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -356,6 +356,17 @@ public function delete_item( $request ) {
 			);
 		}
 
+		/**
+		 * Fires after a widget is deleted via the REST API.
+		 *
+		 * @since 5.8.0
+		 * @param string           $widget_id  ID of the widget marked for deletion.
+		 * @param string           $sidebar_id ID of the sidebar the widget was deleted from.
+		 * @param WP_REST_Response $response   The response data.
```

Comment:
I think let's leave `$response` out. The equivalent hooks in the posts controller don't ever pass along a `WP_REST_Response`.

Reply:
I think it does?

https://github.com/WordPress/wordpress-develop/blob/ad0a55559ae9e4f93b69de6db2233c7793f7c699/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1030

Reply:
Oh. Cool. Alright!